app/Models/Ticket.php

```diff
@@ -10,9 +10,15 @@
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\HasOne;
 use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Str;
 use OpenApi\Attributes as OAT;
 
 #[OAT\Schema(schema: 'Ticket', required: ['title', 'description'], type: 'object')]
+#[OAT\Property(property: 'id', type: 'int', example: 1)]
+#[OAT\Property(property: 'title', type: 'string', example: 'An issue occur with the order #3')]
+#[OAT\Property(property: 'description', type: 'string', example: 'My order dosen\'t include one of the products we request')]
+#[OAT\Property(property: 'customer_id', type: 'int', example: 1)]
```

Comment:
he tenido que poner esto así y eliminar las propiedades porque me daba error al usar el modelo en el evento updated.

Reply:
I had a similiar error we can review this better in the furute for now i will approve an merge the change.